pages/about/AboutContextHandler.inc.php

```diff
@@ -51,6 +51,55 @@ function index($args, $request) {
 
 		$this->setupTemplate($request);
 		$templateMgr = TemplateManager::getManager($request);
+
+		$templateMgr->assign('journal', $context);
```

Comment:
Switched to that for a bunch of templates now. c7972a25086d5e96239f6c308db79a5f5d0882b1